tools/validators/ontology_validator/yamlformat/validator/entity_type_manager.py

```diff
@@ -440,3 +440,20 @@ def _MapTypenamesBySubset(self, types_to_include, field_to_typenames,
       typenames_by_subset[field_set].update(typegroup)
 
     return typenames_by_subset
+
+
+  def GetCompleteFieldSetsOI(self) -> dict():
+    """Returns a mapping of complete field sets to EntityType strings"""
+    #NOTE:This is a temporary implementation meant for development
+    #TODO:Refactor underlying logic to expose field set to entity type maps
+    if self._complete_field_sets_oi == {}:
```

Comment:
realize I'm being a little pedantic here, but this is over-constrained.  It's technically possible for an ontology to exist with no types.  In this case {} is not an accurate indicator of analyze having been run.  

A simple fix for this would be to  initialize   self._complete_field_sets_oi  or   self._typenames_by_subset_oi to None in the constructor, and then set them to {} in the analyze method.